src/System.Configuration.ConfigurationManager/src/System.Configuration.ConfigurationManager.csproj

```diff
@@ -251,8 +252,16 @@
       <Link>Common\System\IO\TempFileCollection.cs</Link>
     </Compile>
   </ItemGroup>
-  <ItemGroup>
+  <ItemGroup Condition="'$(IsPartialFacadeAssembly)' != 'true'">
     <Reference Include="System.Security.Cryptography.ProtectedData" />
+    <Reference Include="System.Security.Permissions" />
```

Comment:
Forgot that we didn't already have that reference. Given that these are all on "internal" APIs we should probably lean towards less dependencies.